examples/data/bio_distr.json

```diff
@@ -303,6 +303,6 @@
         }
     },
     "diameter": {
-        "method": "default"
+        "method": "no_diameters"
```

Comment:
Why not use json's `null` for the None case?

Reply:
no idea :P